src/main/scala/scala/tools/refactoring/implementations/oimports/Region.scala

```diff
@@ -60,9 +60,7 @@ object Region {
   }
 
   private def scanForComments[G <: Global](global: G, source: SourceFile): List[RangePosition] = {
-    val ttb = new TreeToolbox[global.type](global)
-    import ttb.TreeToolboxScanners
-    val scanners = new TreeToolboxScanners
+    val scanners = new TreeToolboxScanners[global.type](global)
```

Comment:
+1, "habit is the deepest law of human nature", I scan other parts to see if they need it (actually there is one global only so no need to narrow it)